### DISCUSSION TITLE
Odadaki yalancı adamlar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#268] [Zeka Sorusu] [answered]
Bir odada oniki adam var. Bunların bir bölümü daima yalan söylüyor, kalanlar daima doğruyu söylüyor. İçlerinden biri şöyle diyor: “Bu odada dürüst tek bir kişi yok”, İkinci adam: “Bu odada birden fazla dürüst kişi yok”. Üçüncü adam: “Bu odada 2’den fazla dürüst kişi yok.” Vb On ikinci adam “Bu odada 11 den fazla dürüst kişi yok.” Odada kaç dürüst kişi var?

**Comment by AhmetEmirErsoz** [comment]
Odada hiç dürüst kişi yoktur

**Comment by adasue** [comment, marked as answer]
Birinci kişiyi yalan söylüyor almalıyız çünkü odada en az 1 doğru ve en az 1 yalan söyleyen kişi olmalı. İkinci adam yalan söylüyor olmalı çünkü eğer doğruyu söyleseydi ikiden sonraki adamların söylediklerini de doğru almamız gerekecekti ve o zaman da birden fazla dürüst kişi olacaktı. Eğer devam edersek 7'inci adama kadar ondan önceki herkesi yalancı olarak almalıyız. 7'inci adam da "Bu odada 6'dan fazla dürüst kişi yok." diyecek. 8'inci adama geçtiğimizde ise "Bu odada 7'den fazla kişi yok." diyecek ve bu da doğru bir ifade o nedenle 7'inci adam dahil 7'inci adamdan sonraki herkesi dürüst olarak aldığımızda **toplamda 6 dürüst kişi oluyor**.
